examples/test_data_gen/src/construct_test_data_flow/utils.py

```diff
@@ -0,0 +1,115 @@
+import json
+
+import numpy as np
+import numpy.testing as npt
+from numpy.random import default_rng
+from promptflow.tools.aoai import chat as aoai_chat
+from promptflow.tools.openai import chat as openai_chat
+
+from promptflow.connections import AzureOpenAIConnection, OpenAIConnection
+
+
+class QuestionType:
+    SIMPLE = "simple"
+    REASONING = "reasoning"
+    CONDITIONAL = "conditional"
+    # MULTI_CONTEXT = "multi_context"
+
+
+class ValidateObj:
+    QUESTION = "question"
+    TEXT_TRUNK = "text_trunk"
+    GROUNDTRUTH = "ground_truth"
+
+
+def llm_call(connection, model, prompt, response_format="text"):
```

Comment:
response_format should be a node parameter.
cause gpt4 0613 cannot work with json